_includes/methodology.html

```diff
@@ -0,0 +1,26 @@
+<section class="flex flex-row md:gap-5 p-2 md:p-10 w-full bg-gray-300 object-cover text-white">
+  <div class="w-full md:w-1/2 p-10 relative p-2">
+    <button class="bg-white rounded-[4px] pt-2 pr-3 pb-2 pl-3 text-lg text-black px-6 py-3 font-medium font-sans mb-8">
```

Comment:
There's no need to use a button if it's not clickable. We can change it to another tag.